externalPackages/FMAToolbox/IO/LoadParameters.m

```diff
@@ -147,15 +163,15 @@
 parameters.nElecGps = length(p.anatomicalDescription.channelGroups.group);
 parameters.ElecGp = p.anatomicalDescription.channelGroups.group;
 parameters.HiPassFreq = 500; % default hi-pass for klusta-3.0 w/ intan data
-parameters.lfpSampleRate = str2num(p.fieldPotentials.lfpSamplingRate);
+parameters.lfpSampleRate = parameters.rates.lfp;
```

Comment:
what's the difference between these two? Is this going to break anything?

Reply:
Answering two questions:

Does this line break things: No.  It takes from the previous changes I made.  This part is just the backcompatibility part of the sessionInfo in fact and now I'm just letting changes and discoveries above be reflected in the backcompatible part.  The previous line was redundant with another call higher up, but neither allowed reading of user-input LPF in neuroscope.  Now the first one checks for user input, this one at 166 takes the out put of that.  See next:

Higher up changes: The old parameters.lfpSampleRate came from some un-changeable default region.  I found where the user-made changes are reflected in the .xml and I now use that if it exists.  If it doesn't exist I revert to previous behavior.  This line 166 thing allows the "backcompatiability" part of the sessionInfo to reflect the work done higher up, rather than just making the same default mistake.

Reply:
Awesome, thanks!